### PR TITLE
Add spec and webextension-polyfill

### PR DESCRIPTION
**Pull request by freaktechnik**
Mozilla created a polyfill to add the `browser` namespaced promise-based API to Chrome and Opera.

It also seems like the spec should be linked here, since it is what this is all about.

### CODE REVIEW

**Comment by bfred-it** [comment]
Thanks! I think I'd like the polyfill a bit higher, just below mozilla/web-ext, using the name `mozilla/webextension-polyfill` to highlight its maintainer.

**Comment by freaktechnik** [comment]
So you'd want it in the tools instead of the libraries section? Did you mean below the other promise polyfill?

**Comment by freaktechnik** [comment]
I just noticed that I have two more links I'd suggest to add to examples:
 - https://github.com/mdn/webextensions-examples
    Very comprehensive collection of extension. I guess they're also linked from MDN, but it is worth linking the repo IMO
 - https://github.com/Standard8/example-webextension
    Does both unit testing using stubs and integration testing via webdriver.

**Comment by freaktechnik** [comment]
Ah, and one more: https://github.com/rpl/create-webextension for Boilerplates

**Comment by bfred-it** [comment]
> So you'd want it in the tools instead of the libraries section? Did you mean below the other promise polyfill?

Ah sorry I didn't notice they weren't in the same category. I guess it's fine there then.

> https://github.com/mdn/webextensions-examples
> Very comprehensive collection of extension. I guess they're also linked from MDN, but it is worth linking the repo IMO

👍

> https://github.com/Standard8/example-webextension
> Does both unit testing using stubs and integration testing via webdriver.

👎 mdn's examples are plenty

> Ah, and one more: https://github.com/rpl/create-webextension for Boilerplates

👍

File: README.md [hunk 1]
```diff
@@ -76,6 +76,7 @@ Apps that help you manage your extensions.
 
 ## Boilerplates
 
+- [create-webextension](https://github.com/rpl/create-webextension) - Yarn webextension generator.
```
Comment (bfred-it) [inline README.md]: -> WebExtension

**Pull request merged**